### DISCUSSION TITLE
1.2.0

### DISCUSSION THREAD

**Discussion by benhillis** [microsoft/WSL#9914] [General]
*  Revert to the 5.15.90.1 version of the kernel while we debug a few issues.

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WSL/releases/tag/1.2.0'>1.2.0</a>.</em>

**Comment by josemtella** [comment]
regresión port proxy (1.1.3)

**Reply by benhillis** [reply]
Fixed in 1.2.1

**Comment by elsaco** [comment]
Maybe the issues are because the codename of 6.1 Linux kernel is `Hurr durr I'ma ninja sloth` 😀 See `Makefile` for details.

**Comment by scaryrawr** [comment]
There's a regression on the user runtime-dir with this release.

On `WSL version: 1.1.6.0` with systemd enabled I get:

```txt
/run/user/1000
```

But on this release it does not get mounted.

**Reply by benhillis** [reply]
This is fixed in 1.2.1

**Comment by wynnw** [comment]
This broke our dev environments. We are on amazonlinux2 with systemd enabled, and systemctl no longer works and fails with this error:

`$  sudo systemctl status
Failed to get D-Bus connection: No such file or directory
`

and you can't start any services, etc. This is a different failure than the 1.1.6.0 release that broke docker for us, but seems systemd/cgroups releated.

**Reply by benhillis** [reply]
Could you please file an issue with repro steps?

**Reply by wynnw** [reply]
1.2.5 fixed that issue for me. Sorry I never got around to making an issue.

**Comment by cmalcolmch** [comment]
Once again, my version of WSL has auto updated to a Pre-release version.
I did not type wsl --update

> wsl --version
WSL version: 1.2.0.0
Kernel version: 5.15.90.1
WSLg version: 1.0.51
MSRDC version: 1.2.3770
Direct3D version: 1.608.2-61064218
DXCore version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows version: 10.0.22000.1696

I am also running Windows 11 🤷‍♀️ but the wsl --version is saying Windows 10.

**Comment by elsaco** [comment]
@cmalcolmch does your `Microsoft Store` app has `App updates` enabled in `Settings`? That setting `Update apps automatically`, WSL included.

**Reply by cmalcolmch** [reply]
Hi @elsaco

Thanks for the reply.

In the Microsoft Store.
App updates
Update apps automatically
On

So here's my question.
https://github.com/microsoft/WSL/releases
Has Tags
1.2.3 Pre-release
1.1.3 Latest

Does the App Store take any notice of the tags and only update to the Latest release.
or
does it it just pick up any Pre-release ?

I would like to stay on a stable release path and not have the latest and greatest Pre-release, that may break things.

Thanks

**Reply by jjpvdpoel** [reply]
I've already opened this issue ( #9970 )
Microsoft likes to give versions the latest release status and revert that to pre-release later, because of bugs.
Meanwhile your app store did the update and now you're stuck with a buggy pre-release.
So turn of auto update and hope microsoft will do better in the future (like make it open source).
